rllib/core/learner/learner.py

```diff
@@ -849,14 +851,16 @@ def compute_losses(
         """Computes the loss(es) for the module being optimized.
 
         This method must be overridden by MultiRLModule-specific Learners in order to
-        define the specific loss computation logic. If the algorithm is single-agent
-        `compute_loss_for_module()` should be overridden instead. If the algorithm uses
-        independent multi-agent learning (default behavior for multi-agent setups), also
-        `compute_loss_for_module()` should be overridden, but it will be called for each
-        individual RLModule inside the MultiRLModule.
+        define the specific loss computation logic. If the algorithm is single-agent,
+        only `compute_loss_for_module()` should be overridden instead. If the algorithm
+        uses independent multi-agent learning (default behavior for RLlib's multi-agent
+        setups), also only `compute_loss_for_module()` should be overridden, but it will
+        be called for each individual RLModule inside the MultiRLModule.
```

Comment:
Maybe add a point for when to specifically override `compute_losses` instead of `compute_loss_for_module`